File: tasks/isPermutation.cpp
bool isPermutation(int n, std::vector<int> s) {
  if (s.size() != n)
    return false;

  std::sort(s.begin(), s.end());
  for (int i=0; i<s.size(); i++) {
    if (s[i] != i + 1)
      return false;
  }

  return true;
}

File: tasks/isPermutation.java
boolean isPermutation(int num, int[] ar) {
  int n=ar.length;
  int x=num;
  int cnt=0;
  for(int i=0; i<n; )
  {
    if(ar[i]==x)
    {
      cnt++;
      x--;
      i=0;
      continue;
    }
    i++;
  }
  if(cnt==num)
    return true;
  else
    return false;
}
